db/models/aggregate_functions.py

```diff
@@ -186,6 +186,11 @@ def update_application_and_related_form(
     application = ApplicationsMethods.get_application_by_id(application_id)
     application.last_edited = func.now()
     form_sql_row = FormsMethods.get_form(application_id, form_name)
+    if form_sql_row.name == "project-information":
+        fields_array = form_sql_row.json[0]["fields"]
+        for key in fields_array:
+            if key["key"] == "project-name":
+                application.project_name = key["answer"]
```

Comment:
Could we use `filter` here instead for `for` iteration?

Reply:
we're only using this for loop once here and nowhere else in the codebase, so would it be unnecessary to separate this out into a sperate function to use in a filter?